stdlib/http/src/main/java/org/ballerinalang/net/http/exceptions/WebSocketException.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ * <p>
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.ballerinalang.net.http.exceptions;
+
+import org.ballerinalang.jvm.util.exceptions.BallerinaException;
+
+/**
+ * BallerinaException that could occur in Artemis connector.
```

Comment:
Shall we change the comment to "BallerinaException that could occur in WebSocket"